dojo/api_v2/permissions.py

```diff
@@ -453,9 +453,8 @@ def has_permission(self, request, view):
             )
         elif engagement_id:
             # engagement_id doesn't exist
-            raise serializers.ValidationError(
-                "Engagement '%s' doesn''t exist" % engagement_id
-            )
+            msg = f"Engagement '{engagement_id}' doesn''t exist"
```

Comment:
Might as well clean these up while we're looking at them now

```suggestion
            msg = f"Engagement '{engagement_id}' doesn't exist"
```